crt_portal/cts_forms/views.py

```diff
@@ -590,22 +590,43 @@ def data_view(request):
         display_name = name.replace("_", " ").capitalize()
         assignment = FileGroupAssignment.objects.filter(analytics_file=notebook.pk).first()
         group = None
+        sections = []
         if assignment:
             group = DashboardGroup.objects.filter(pk=assignment.dashboard_group.pk).first().header
+            section_objects = assignment.show_only_for_sections.all()
```

Comment:
nit: objects may come back from the database in arbitrary order. You may want to specify an order for consistent results